packages/urls/src/urls.ts

```diff
@@ -29,12 +29,15 @@ export const HELP_CENTER_DRY_RUN_T1_URL =
     'https://trezor.io/learn/a/test-recovery-seed-on-trezor-model-one';
 export const HELP_CENTER_DRY_RUN_TT_URL =
     'https://trezor.io/learn/a/test-recovery-seed-on-trezor-model-t';
+export const HELP_CENTER_DRY_RUN_TR_URL =
+    'https://trezor.io/learn/a/test-recovery-seed-on-trezor-model-r';
```

Comment:
There is no such page, right? But I see that trezor.io will return 200 status anyway, so tests pass. Nevermind, we just have to request those new pages. Done here: https://www.notion.so/satoshilabs/Model-R-equivalents-for-articles-linked-from-Suite-2dd87af7d2444bb8a51c1240848ed1d6

Reply:
Okay thanks, I have it in my notion doc